eng/SourceBuild.props

```diff
@@ -13,6 +14,19 @@
       WorkingDirectory="$(InnerSourceBuildRepoRoot)" />
   </Target>
 
+  <!--
+    Init submodules - temporarary workaround for https://github.com/dotnet/sourcelink/pull/653
```

Comment:
https://github.com/dotnet/sourcelink/pull/653 was fixed by @tmat in 2018. perhaps file a new issue?

Reply:
We are tracking the backport of this patch with https://github.com/dotnet/source-build/issues/2708.